### ISSUE TITLE
Apocrine metaplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10074]
**Preferred term label:**
Apocrine metaplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0334036

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Intraductal papilloma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
same comment as https://github.com/obophenotype/human-phenotype-ontology/issues/10071

**Comment by pnrobinson** [comment]
@MickeySegal 
adding as
Mammary ductal apocrine metaplasia
HP:0020329
Def: A biopsy finding in breast tissue of enlarged epithelial cells with abundant granular eosinophilic cytoplasm that can show apical snouting. Metaplastic apocrine cells within the breast are similar to normal axillary, areolar and perineal apocrine cells, having apocrine secretions and identical staining reactions. One of the reasons, however, that these are often not regarded as normal components is that examples of gradual metaplastic change from the normal cuboidal epithelium can be seen and cells with intermediate features can also be discerned on careful examination, suggesting that these are true metaplasia.
